### PR TITLE
[KO] Retranslate the Ranking_Criteria/osu!taiko article

### PR DESCRIPTION
**Pull request by libneko**
Re-worked all which was different with the original document.

---

### CODE REVIEW

File: wiki/Ranking_Criteria/osu!taiko/ko.md [hunk 1]
```diff
@@ -1,61 +1,163 @@
...
+- **태고 템플릿 배경:** 본래 태고의 달인과 비슷한 느낌이 되도록 하는 배경 이미지. 보통 검은색 라인과 함께 흰색 글씨로 아티스트, 타이틀을 표기합니다.
+
+## 일반
+
+### 규칙
+
+모든 **규칙**은 가이드라인이 **아니므로**, **어떠한 상황**에서도 **어기면 안됩니다**.
+
+- **각 노트는 이전 및 이후에 올 노트와 명확히 구분이 가능한 수 있는 색상을 가지고 있어야 합니다.**
+- **모든 음은 그것이 표현하려고 하는 음악 레이어나 노래가 제공하는 레이어를 강화하기 위한 레이어 유닛, 아니면 매퍼에 의해 즉흥적으로 만들어진 추가적인 레이어 유닛인지 분명하게 지정할 수 있어야 합니다.** 없는 속도 변화를 만들거나, 일반적인 음악의 움직임과 모순되거나, 노래의 강도를 잘못 해석하는 방식으로 즉흥 매핑을 하지 마세요. 즉흥 매핑은 노래의 레이어를 강화하거나, 새로운 레이어를 추가해야 합니다. 그렇지 않으면, 노래와의 관계를 포기하는 것이 되므로 리듬 게임의 주된 목적과는 모순되게 됩니다.
```
Comment (libneko) [inline wiki/Ranking_Criteria/osu!taiko/ko.md]: There was an opinion that It's hard to understand the meaning of overall sentence.

File: wiki/Ranking_Criteria/osu!taiko/ko.md [hunk 2]
```diff
@@ -1,61 +1,163 @@
...
+### 가이드라인
+
+가이드라인은 **예외적인** 상황에서 위반될 수 있습니다. 이러한 예외적인 상황은 가이드라인을 위반한 이유와, 왜 그것이 만들어진 것의 전반적인 질을 저해하지 않았는지 철저한 설명으로 보장되어야 합니다.
+
+- **AiMod를 주의해서 사용해주세요**. 기본 슬라이더 속도와 관련된 버그를 일으키는 것으로 알려져 있습니다.
+- **태고 플레이 필드가 배경의 중요한 부분을 가리지 않게 하세요.** 만약 가려진다면 메모장을 이용해 `[Events]`헤더의 `0,0,"배경파일.확장자",0,0` 행의 마지막 0을 변경해보세요. 양수 값은 배경을 아래로 내리며, 음수 값은 위로 올립니다.
+- **슬라이더 속도 변경을 사용 중이라면, 곡의 속도 변화와 일치하게 만드세요.** 즉, 잔잔한 부분에서 속도를 높이거나 빠른 느낌의 부분에서 속도를 늦추지 말라는 소리입니다.
+- **가변 스냅이 포함된 섹션에서 부드러운 변속을 사용하지 마세요.** 그렇게 하면 스냅 된 노트에 가독성에 영향을 미치므로, 속도 변화를 최소한으로 하여 오버랩을 방지하세요.
+- **이미 오버랩된 패턴 내에서 급격한 변속은 피해주세요 (예: 1/4 스트림).** 부드러운 변속을 이럴 때 사용하여 패턴을 인식할 수 있도록 해야 합니다.
+- **쓸데없이 시각적인 방해 없이 색 구분을 하기 쉽게 심한 오버랩은 피해주세요.** 오버랩은 그 지점의 노래의 속도나 박자를 맞추기 위해 노트를 찍는 것이 정당화될 수 있는 경우에만 이루어져야 합니다.
```
Comment (libneko) [inline wiki/Ranking_Criteria/osu!taiko/ko.md]: This sentence needs to be improved on grammar.

File: wiki/Ranking_Criteria/osu!taiko/ko.md [hunk 3]
```diff
@@ -1,61 +1,163 @@
-osu!taiko Ranking Criteria
-====================================
+osu!taiko 
+=============
+
+_Main page: [Ranking Criteria](/wiki/Ranking_Criteria)_
```
Comment (TPGPL) [inline wiki/Ranking_Criteria/osu!taiko/ko.md]: Translate "Main page"

Comment (libneko) [reply wiki/Ranking_Criteria/osu!taiko/ko.md]: Okay.

**Pull request merged**